plugins/source/aws/resources/services/iam/last_accessed_jobs.go

```diff
@@ -0,0 +1,108 @@
+package iam
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+	"github.com/aws/aws-sdk-go-v2/service/iam/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+type Job struct {
+	Arn   string `json:"arn"`
+	JobId string `json:"job_id"`
+}
+
+func userLastAccessedJobs() *schema.Table {
+	return &schema.Table{
+		Name:        "aws_iam_user_last_accessed_jobs",
+		Description: `https://docs.aws.amazon.com/IAM/latest/APIReference/API_GenerateServiceLastAccessedDetails.html#API_GenerateServiceLastAccessedDetails_Example_1_Response`,
+		Resolver:    fetchUserLastAccessedJobs,
+		Transform:   transformers.TransformWithStruct(&Job{}, transformers.WithPrimaryKeys("Arn")),
+		Multiplex:   client.ServiceAccountRegionMultiplexer("iam"),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(true),
+		},
+		Relations: []*schema.Table{userLastAccessedDetails()},
+	}
+}
+
+func roleLastAccessedJobs() *schema.Table {
+	return &schema.Table{
+		Name:        "aws_iam_role_last_accessed_jobs",
+		Description: `https://docs.aws.amazon.com/IAM/latest/APIReference/API_GenerateServiceLastAccessedDetails.html#API_GenerateServiceLastAccessedDetails_Example_1_Response`,
```

Comment:
```suggestion
		Description: `https://docs.aws.amazon.com/IAM/latest/APIReference/API_GenerateServiceLastAccessedDetails.html`,
```

I guess we don't want to link to the example response specifically?

Reply:
Good point I missed that doc